docs/design/milestone_1.md

```diff
@@ -0,0 +1,320 @@
+
+# Criticality Score Revamp: Milestone 1
+
+- Author: [calebbrown@google.com](mailto:calebbrown@google.com)
+- Updated: 2022-04-29
+
+## Goal
+
+Anyone can reliably generate raw signal data using the `criticality_score`
+GitHub project.
```

Comment:
Should we explicitly mention generating the default criticality list as well?

Reply:
I think this is covered in the third paragraph in this section.

Reply:
I think it's a bit vague what this would entail for Milestone 1. Explicitly mentioning that we will provide the ability to generate this list of rankings using the existing algorithms may make this a lot clearer.

Reply:
PTAL. Hopefully this is more explicit.

Reply:
LGTM, thanks!